docs/Installation.md

```diff
@@ -322,7 +325,20 @@ from wich the GUI is served.
 **NOTE:** Don't forget to apply the changes to the `index.html` and Web
 server configurartion after each updates as thoses files will be overwritten.
 
--------
+
+## Change default language
+
+To change the default language update the Apache configuration, locate the last `RewriteRule`:
```

Comment:
In your apache configuration if you have used the one provided as example.

Reply:
Please include that information into the document.

Reply:
You mean `zonemaster.conf`, don't you? I do not think it is obvious here. To make it clear it could say:
```
To change the default language update the Apache configuration in `zonemaster.conf` that was installed
above. Locate the last `RewriteRule`:
```